scripts/test-integration.bat

```diff
@@ -80,6 +80,13 @@ mkdir %GITWORKSPACE%
 call "%INTEGRATION_TEST_ELECTRON_PATH%" %GITWORKSPACE% --extensionDevelopmentPath=%~dp0\..\extensions\git --extensionTestsPath=%~dp0\..\extensions\git\out\test %API_TESTS_EXTRA_ARGS%
 if %errorlevel% neq 0 exit /b %errorlevel%
 
+echo.
```

Comment:
same as above

Reply:
`call npm run test-extension` is the more modern way to run integration tests with all the UI functionality. @connor4312 would know more though